recipe/jwt/jwtmodels/recipeInterface.go

```diff
@@ -18,7 +18,7 @@ package jwtmodels
 import "github.com/supertokens/supertokens-golang/supertokens"
 
 type RecipeInterface struct {
-	CreateJWT *func(payload map[string]interface{}, validitySeconds *uint64, userContext supertokens.UserContext) (CreateJWTResponse, error)
+	CreateJWT *func(payload map[string]interface{}, validitySeconds *uint64, userContext supertokens.UserContext, useStaticSigningKey *bool) (CreateJWTResponse, error)
```

Comment:
AFAIK everywhere else, the `userContext` is the last param, I don't think we should break this pattern. Would it cause issues?